### PR TITLE
Drop Ruby 1.9.x support

### PR DESCRIPTION
**Pull request by ixti**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,12 +4,12 @@ env:
...
 language: ruby
 rvm:
-  - 1.9.3
   - 2.0.0
   - 2.1
   - 2.2
   - 2.3.0
-  - jruby
+  - jruby-9.0.0.0
+  - jruby-9.0.1.0
```
Comment (tarcieri) [inline .travis.yml]: 9.0.5.0 here perhaps?

Comment (ixti) [inline .travis.yml]: Added 9.0.{4,5}.0 as well... But there are couple of specs failing on 9000+ in general o_O

Comment (ixti) [inline .travis.yml]: In fact it fails on `9.0.{0,1,4}.0` only. `9.0.5.0` works fine indeed.

Comment (tarcieri) [inline .travis.yml]: I think it's fine to just test `9.0.5.0`. Those are just patchlevel changes.

**Comment by ixti** [comment]
@tarcieri done. As we test JRuby 9.0.5.0 only I have chnaged `JRuby 9000+` support declaration in readme to `9.0.5.0`

**Comment by tarcieri** [comment]
Cool. I think I'd like to get #298 squared away first and do one more 1.x release before merging this though.

**Comment by tarcieri** [comment]
:shipit:

**Comment by tarcieri** [comment]
@ixti @sferik @zanker FYI to you all, master is now version 2.x of the gem, which requires Ruby 2.0+.

`1-x-stable` is the support branch for doing 1.x bugfixes. Hopefully there won't be that many.

**Comment by tarcieri** [comment]
I didn't do a gem release yet. @ixti you want to do the honors?

**Comment by tarcieri** [comment]
@ixti actually, before doing a 2.x release do you want to weigh in on #297 ?

**Comment by tarcieri** [comment]
2.0.0 release tracking issue here: #321

**Pull request merged**